test-app/src/components/bns.tsx

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 
 import { Box, styled } from 'leather-styles/jsx';
 
+/** TODO 4370 - Delete this as the link is broken ???? */
+/** @deprecated */
```

Comment:
This opens a link to `https://btc-website.tintashlabs.com/` which fails as its not secure. 

If I go to the `http://` it gives a 403 error